include/cuda/std/detail/libcxx/include/support/atomic/atomic_cuda.h

```diff
@@ -121,13 +121,43 @@ const volatile _Tp* __cxx_get_underlying_device_atomic(__cxx_atomic_base_heterog
 }
 
 template <typename _Tp>
-using __cxx_atomic_small_to_32 = typename conditional<is_signed<_Tp>::value, int32_t, uint32_t>::type;
+using __cxx_atomic_small_to_32 =
+    typename conditional<is_signed<_Tp>::value,
+        int32_t,
+        uint32_t
+    >::type;
```

Comment:
We have a helper alias for this now, so that we do not have to bother with the `meow_t` not being available in C++11
```suggestion
using __cxx_atomic_small_to_32 = __conditional_t<is_signed<_Tp>::value, int32_t, uint32_t>;
```